bin/.preload.sh

```diff
@@ -7,7 +7,7 @@ DFAULTCONFIG="conf/config.properties"
 CUSTOMCONFIG="data/settings/customized_config.properties"
 LOGCONFIG="conf/logs/log-to-file.properties"
 STARTUPFILE="data/startup.tmp"
-DFAULTXmx="-Xmx800m";
+DFAULTXmx="1024m";
```

Comment:
`DFAULTXmx` is one of the command line arguments for `java` to run Loklak. It's not just a value like what's on `config.properties`.